### ISSUE TITLE
Overlapping the navbar[BUG]

### ISSUE THREAD

**Issue by SubhashThenua** [omegaup/nonprofit-web#77]
**Current Behavior**
When the home page window size is minimized the navigation bar is getting overlapped with the logo
******************************************************************************
**Expected Behavior**
The nav bat shouldn't overlap.
Describe what you were originally trying to do and how you expected the platform to behave.
When the window is minimized the navigation bar need to be convert into menu bar for better view

Relevant details about the environment in which you encountered this bug:
```
1. Browser name (e.g., Chrome 39, node.js 5.4): Chrome
2. Operating System and version (desktop or mobile): Windows 11
3.Link to the site where this issue occurs:https://www.omegaup.org/
4.Screenshot:


```

![image](https://github.com/omegaup/nonprofit-web/assets/100484672/177f6a31-6dcb-41fa-a980-85d0152ee784)

**Comment by Neelanchal17** [comment]
Yeah, found the same issue, I was about to create a new issue for this, but found this post, I would like to ask the maintainers if I could fix this issue?
